spec/controllers/collection_participants_controller_spec.rb

```diff
@@ -400,7 +406,10 @@
       context "where the users to be added haven't yet applied to the collection" do
         it "creates new participants with the member role and redirects" do
           get :add, params
-          it_redirects_to collection_participants_path(collection)
+          # We use it_redirects_to_simple as it_redirects_to
+          # checks that there are no extra error,notice flash messages set
```

Comment:
There should be a space after the comma here. (Also, it might just be me, but I think "because" might be clearer than "as" here -- I initially thought this meant `it_redirects_to_simple` was acting as an alias for `it_redirects_to`.)